BeatSaberHTTPStatus/Plugin.cs

```diff
@@ -353,11 +355,17 @@ public class Plugin : IPlugin {
 					statusManager.EmitStatusUpdate(ChangedProperties.PerformanceAndNoteCut, "noteMissed");
 				}
 			}
+		}
+
+		public void OnNoteWasCut2(NoteController noteController, NoteCutInfo noteCutInfo) {
```

Comment:
This is a very undescriptive name. How does it differ from `OnNoteWasCut`? Also, why is this one necessary and what order are they called in?